tests/integration/support/steps.go

```diff
@@ -134,7 +135,8 @@ func (c *ScenarioCtx) TheResultShouldBeBoolean() error {
 
 // TheResultShouldMatchList represents a step in the feature file
 func (c *ScenarioCtx) TheResultShouldMatchList(list string) error {
-	if c.apiResponse.Result == list {
+	expectedList := strings.Split(list, ",")
+	if actualList := c.apiResponse.Result.([]string); compareStringSlice(expectedList, actualList) {
```

Comment:
nit: Here you should use the `ok` param 

if `actualList, ok := c.apiResponse.Result.([]string); ok { ... }`